packages/design-system-docs/src/pages/components/DateField/datefield.example.html

```diff
@@ -33,6 +37,7 @@
         name="birthdate[day]"
         autocomplete="bday-day"
         aria-describedby="dob-datefield"
+        value="31"
       />
     </div>
     <span class="ds-c-datefield__separator">/</span>
```

Comment:
Add `ds-c-field--error` to line 50 class to keep it consistent with the inverse example